File: ex34.c
#include <stdio.h>
 
int main(int argc, char *argv[]){
  //int numeros[5] = {4, 5, 7, 10, 18};
  int numeros[5];
  int maior=0;
  int menor=0;
 // int menor = numeros[5];


  printf("Digite os 10 numeros: ");
  for(int i=0; i<5; i++){
   if (numeros[i]==0) maior = numeros[i];
    if (numeros[i] > maior) maior = numeros[i];

      scanf("%i", &numeros[5]);
  }




{
  printf("o maior numero é o : ");
      printf("%i", maior);
  }
  

  return 0;
}

File: python/ex34.py
print('imprimir o maior de 10 numeros')
 
numeros = []
maior=0


for i in range(0, 10, 1):

     n = int(input('Digite os numeros: '))
     numeros.append(n)

     if (numeros[i] > maior):
         maior = numeros[i]
        
print('O numero maior é o:')
for i in range(0, 1, 1):
      
    print(maior)